core/src/main/java/lucee/runtime/functions/string/StringReduce.java

```diff
@@ -0,0 +1,31 @@
+package lucee.runtime.functions.string;
+
+import lucee.runtime.PageContext;
+import lucee.runtime.exp.FunctionException;
+import lucee.runtime.exp.PageException;
+import lucee.runtime.ext.function.BIF;
+import lucee.runtime.functions.closure.Reduce;
+import lucee.runtime.op.Caster;
+import lucee.runtime.type.UDF;
+import lucee.runtime.type.util.StringListData;
+
+public class StringReduce extends BIF {
+	private static final long serialVersionUID = -2153555241217815037L;
+
+	@Override
+	public Object invoke(PageContext pc, Object[] args) throws PageException {
+		if (args.length < 2) {
+			throw new FunctionException(pc, "StringSome", 2, 3, args.length);
+		}
+		return call(pc, Caster.toString(args[0]), Caster.toFunction(args[1]), args[2]);
+	}
+
+	public static Object call(PageContext pc, String inputString, Object value, Object initialValue) throws PageException {
+		StringListData stringList = new StringListData(inputString, "", false, false);
```

Comment:
can you use the same name pattern as we have for ArrayReduce and StructReduce so
`public static Object call(PageContext pc, String str, UDF udf, Object initValue)`